client/reader/stream/index.jsx

```diff
@@ -157,8 +159,8 @@ class ReaderStream extends React.Component {
 		KeyboardShortcuts.on( 'like-selection', this.toggleLikeOnSelectedPost );
 		KeyboardShortcuts.on( 'go-to-top', this.goToTop );
 		window.addEventListener( 'popstate', this._popstate );
-		if ( 'scrollRestoration' in history ) {
-			history.scrollRestoration = 'manual';
+		if ( 'scrollRestoration' in window.history ) {
```

Comment:
should we guard this against a node-like environment by checking to see if `window` is an object?

Reply:
It's `componentDidMount`, therefore it's never called in SSR.

The methods that need to be SSR safe are the constructor, `componentWillMount` and `render`. All others are run only in interactive environment, i.e., only in browser.